fecgov/fec-cms
# Feature/3600 presidential map 4

## Summary

- Resolves #3600
Many things from the MoSCoW doc. This should be considered the release beta.

## Impacted areas of the application
- Application-wise, only this map and WCCF may be affected.
- Technically any other data page with accordions could be affected but shouldn't (If the other data pages are impacted, it would be in that other accordions would be limited so only one item is open at a time and they wouldn't close correctly.)

## Screenshots
![image](https://user-images.githubusercontent.com/26720877/76862100-f3bb9880-6833-11ea-83c7-9f347cede006.png)

## Related PRs
None active

## How to test
I added basically everybody as a reviewer; feel free to remove or add whomever

Either pull, build, and run as normal, or
*Just check Feature*

- I didn't check off anything in the MoSCow doc so if one of you could, great! (Seemed like grading my own paper)
- The Musts should be taken care of (except the 'loading' visual feedback)
- Pretty sure most of the Shoulds and Coulds are in, too
- Check against different browser widths from as small as you can go (400-500px) up to normal large (1200?)

____
